clients/ui/frontend/src/app/utils/useK8sWatchResourceList.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  K8sModelCommon,
+  K8sResourceCommon,
+  WatchK8sResource,
+  useK8sWatchResource,
+  WebSocketOptions,
+} from 'mod-arch-shared';
+import React from 'react';
+import { CustomWatchK8sResult } from '~/app/k8sTypes';
+
+const useK8sWatchResourceList = <T extends K8sResourceCommon[]>(
```

Comment:
@lucferbux This is wrapper we created for `websocket` in midstream, as we are porting groups and in midstream groups uses websocket - should we port this as well?